src/server/scripts/Spells/spell_item.cpp

```diff
@@ -3717,33 +3717,34 @@ class spell_item_taunt_flag_targeting : public SpellScript
 
 enum MirrensDrinkingHat
 {
-    ITEM_LOCH_MODAN_LAGER       = 23584,
-    ITEM_STOUTHAMMER_LITE       = 23585,
-    ITEM_AERIE_PEAK_PALE_ALE    = 23586
+    SPELL_LOCH_MODAN_LAGER      = 29827,
+    SPELL_STOUTHAMMER_LITE      = 29828,
+    SPELL_AERIE_PEAK_PALE_ALE   = 29829
 };
 
 // 29830 - Mirren's Drinking Hat
```

Comment:
Cosmetic: I know this is not consistent in the scripts, but it would be nice to see this comment as a start indicator above the enum list. (Please move it above `enum MirrensDrinkingHat` ?)